app/react/Forms/components/Geolocation.js

```diff
@@ -39,11 +44,30 @@ export default class Geolocation extends Component {
   }
 
   latChange(e) {
+    if (!e.target.value) {
+      this.setState({ currentLatitude: e.target.value });
+      return;
+    }
+
+    let latitude = parseFloat(e.target.value);
+
+    // The latitude is a value between -90 degrees and 90 degrees
+    // The page breaks if the map is refreshed with a latitude out of range
```

Comment:
We try to avoid comments (unless really indispensable) on the code.  These are usually better described in the test that proves this concept.

Imagine a condition where this changes, and someone changes the value of the range to -100 to 100, but then forgets to update the comment?  Then they would get disconnected (or if text is inserted between the comment and the code).

This can also happen in the tests, but it provides less clutter.

Reply:
Comments removed. Thank you!